src/gui/wallet.go

```diff
@@ -499,7 +483,8 @@ func walletNewAddresses(gateway *daemon.Gateway) http.HandlerFunc {
 		}
 
 		if err := Wg.SaveWallet(wltID); err != nil {
-			wh.Error500(w, "")
+			logger.Error("save wallet failed when generate new addresses: %v", err)
```

Comment:
The`logger.Error("save wallet failed when generate new addresses:", err)` will log the message as: `2017/06/17 13:11:29 save wallet failed when generate new addresses:%!(EXTRA *errors.errorString=invalid address)`. The `"github.com/op/go-logging"` version we current use is too old, must have `format` string. We can create an issue to update the logger version and update all logs.